data/json/items/fuel.json

```diff
@@ -203,6 +203,30 @@
     "damage": { "damage_type": "heat", "amount": 2, "armor_penetration": 2 },
     "effects": [ "FLAME", "STREAM" ]
   },
+  {
+    "id": "crude_oil",
+    "type": "AMMO",
+    "name": { "str_sp": "crude oil" },
+    "description": "A thick black slurry that can be refined to produce usable products including gasoline or diesel",
+    "category": "fuel",
+    "weight": "1000 mg",
+    "volume": "200ml",
+    "price": 20,
+    "price_postapoc": 50,
+    "count": 250,
+    "stack_size": 200,
+    "phase": "liquid",
+    "looks_like": "diesel",
+    "container": "bottle_plastic",
+    "sealed": false,
+    "symbol": "=",
+    "color": "black",
+    "material": [ "motor_oil" ],
```

Comment:
This definitely should not be useable in engines. Without any distillation it'll wreak havoc on even gas turbines.

Reply:
My bad, forgot to handle that part of the creation of crude_oil, it's fixed now ! Thank you !